exoctk/exoctk_app/app_exoctk.py

```diff
@@ -36,6 +38,21 @@
 app_exoctk.config['CACHE_TYPE'] = 'null'
 app_exoctk.config['SECRET_KEY'] = 'Thisisasecret!'
```

Comment:
Can't be a secret if it's in a public repo 🤔

Reply:
Haha, this has been here forever.